packages/embed/src/utils/url-parameters-transfer.ts

```diff
@@ -0,0 +1,32 @@
+type Params = { [key: string]: string }
+
+type QueryStrings = {
+  [key: string]: any
+}
+
+const urlSearchToParams = (search: string) => {
+  var params: Params = {}
+  if (search !== '' && search !== null) {
+    var vars = search.split('&')
+    for (var i = 0; i < vars.length; i++) {
+      var pair = vars[i].split('=')
+      params[pair[0] as keyof Params] = decodeURIComponent(pair[1])
+    }
+  }
+  return params
+}
+
+export const transferUrlParametersToQueryStrings = (
```

Comment:
Is this still a work in progress PR? I dont see the method actually used anywhere. Please include an example in `demo-html` and a functional test (specify transferable params, visit the demo page with those params in query string and check it is also present in typeform iframe src).

Reply:
Where should I add the example? React, HTML all of them?
Why functional tests? I think this unit test is enough. We just need to check the code, not the UI or the user actions.

Reply:
I think HTML is sufficient. You are right - since there is not UI we dont need visual tests (but still should have functional test). Unit tests confirms the method works correctly. Functional test confirms the function is implemented correctly (you only need to check the iframe src is correct, no need to check how renderer handles it further).